app/controllers/pg_hero/home_controller.rb

```diff
@@ -259,21 +265,37 @@ def tune
       @title = "Tune"
       @settings = @database.settings
       @autovacuum_settings = @database.autovacuum_settings if params[:autovacuum]
+      @citus_enabled = @database.citus_enabled?
     end
 
     def connections
       @title = "Connections"
       @connection_sources = @database.connection_sources
       @total_connections = @connection_sources.sum { |cs| cs[:total_connections] }
-
+      @citus_enabled = @database.citus_enabled?
+      
       @connections_by_database = group_connections(@connection_sources, :database)
       @connections_by_user = group_connections(@connection_sources, :user)
+      if @citus_enabled
```

Comment:
I believe these changes belongs to another PR.

Reply:
Yes they do. I accidentally put them here.